ssl/ssl_local.h

```diff
@@ -1226,8 +1226,8 @@ typedef struct cert_pkey_st CERT_PKEY;
 struct quic_data_st {
     struct quic_data_st *next;
     OSSL_ENCRYPTION_LEVEL level;
-    size_t start;       /* offset into quic_buf->data */
     size_t length;
+    unsigned char data[];
```

Comment:
Sigh.  Hugely not a fan of this.  google zero-length array and see what Dennis Ritchie had to say.

Are we *sure* that this makes things much better?

Reply:
And now I read the discussion in the other PR.  I'm less concerned.  We should document the requirement for var-length-arrays as defined in C9x (whatever year).  Approved.